src/Sulu/Bundle/MediaBundle/DependencyInjection/Configuration.php

```diff
@@ -232,6 +232,7 @@ private function addStorageSection(ArrayNodeDefinition $node)
                         ->scalarNode('path_prefix')->defaultNull()->end()
                         ->scalarNode('version')->defaultValue('latest')->end()
                         ->scalarNode('endpoint')->defaultNull()->end()
+                        ->scalarNode('url')->defaultNull()->end()
```

Comment:
Not sure yet about naming it url maybe we should name it `cdn` or `cdn_url`. @wachterjohannes what do you think?

Reply:
/cc @chirimoya